### PR TITLE
Updated ChangeLog and pod spec for 1.5 release

### PR DESCRIPTION
**Pull request by sid-github**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
 
+## [1.5.0](https://github.com/google/EarlGrey/tree/1.5.0) (10/31/2016)
+
+```
+Baseline: [d46a15f]
+   + [d46a15f]: Update Info.plist / Podspec / Cheatsheet for EarlGrey 1.4.0
+```
+
+### Compatibility
+* Add iOS 10 Support to Travis tests.
```
Comment (tirodkar) [inline CHANGELOG.md]: You should change this to:

Compatibility
* Requires iOS 8 as the minimum deployment target.
* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3. Swift 2.2 is no longer supported.
* Has been tested for support till iOS 10.01 on devices and simulator.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,26 @@
...
+* Update Projects to use new Swift 3.0 proposed syntax.
+
+### Bug Fixes
+* Failing analytics tests fixes.
+* Fix flaky Travis Stopwatch Test.
+* Fix rspec tests broken by ruby update and changing the directory.
+* Safe to invoke failure handlers multiple times within context of a valid test case.
+
+### Internal
+* Improved UIAppStateTracker APIs to allow for ignoring states.
```
Comment (tirodkar) [inline CHANGELOG.md]: Enhancement

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,26 @@
...
+
+### Compatibility
+* Add iOS 10 Support to Travis tests.
+* Update Projects to use new Swift 3.0 proposed syntax.
+
+### Bug Fixes
+* Failing analytics tests fixes.
+* Fix flaky Travis Stopwatch Test.
+* Fix rspec tests broken by ruby update and changing the directory.
+* Safe to invoke failure handlers multiple times within context of a valid test case.
```
Comment (tirodkar) [inline CHANGELOG.md]: Enhancement

File: CHANGELOG.md (hunk 1 above)
Comment (tirodkar) [inline CHANGELOG.md]: Travis & Swift 3.0 are enhancements.
Also link Swift 3.0 to the design doc.

Swift 3.0 [design doc](...)

Comment (sid-github) [inline CHANGELOG.md]: done

Comment (sid-github) [inline CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (sid-github) [inline CHANGELOG.md]: Done

**Pull request merged**